### ISSUE TITLE
'ERROR: Could not retrieve IP' when docker service fails to start

### ISSUE THREAD

**Issue by miabbott** [projectatomic/atomic-devmode#7]
I managed to get my docker daemon in a state where it is failing to start.

Because of this, it appears that the codepath for printing the IP address to the console is failing incorrectly.

The system does have an IP address (I can SSH to it), but the welcome messages suggests otherwise.

![screenshot_rhel-atomic-cloud-7 3 0-1 vm929a_2016-09-29_11 39 17](https://cloud.githubusercontent.com/assets/10236257/18960941/5eaef0e6-8639-11e6-91ca-394471c70646.png)

**Comment by jlebon** [comment]
Yeah, that could be made smarter. But then again, if docker isn't working, you've got bigger problems on your hands. :)
